en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md

```diff
@@ -0,0 +1,318 @@
+# FHIR Connector Example
+
+Given below is a sample scenario that demonstrates how to work with resources in [FHIR](https://www.hl7.org/fhir/overview.html)(Fast Healthcare Interoperability Resources). FHIR specification is a standard for exchanging health care information electronically.The WSO2 EI FHIR connector uses the FHIR RESTFul API to interact with FHIR.
+
+## What you'll build
+
+Given below is a sample API that illustrates how you can connect to a FHIR with the `init` operation and then use the `create`,`readResource`, `readSpecificResourceById`,`delete` and `update` operation to send messages to the FHIR server.
+
+* `<fhir.init>` : It will initialize the FHIR operations.
+* `<fhir.create>` : To creates a new resource in a server-assigned location, use `fhir.create` operation.
+* `<fhir.readResource>` : To accesses the current contents of a resource, use `fhir.readResource` operation.
+* `<fhir.readSpecificResourceById>` : To accesses the specific current contents of a resource, use `fhir.readSpecificResourceById` operation.
+* `<fhir.update>` : To creates a new current version for an existing resource or creates an initial version if no resource already exists for the given id, use `fhir.update` operation.
+* `<fhir.update>` : To removes an existing resource, use `fhir.delete`operation.
```

Comment:
All these should be in the reference documentation so no need to mention here.

Reply:
Removed this part and just mentioned only the content.